mas-foundation/build.gradle

```diff
@@ -206,6 +153,85 @@ artifacts {
     archives sourcesJar
 }
 
+afterEvaluate {
+    apply plugin: 'maven-publish'
+    publishing {
+        publications {
+            release(MavenPublication) {
+                // The coordinates of the library, being set from variables that
+                // we'll set up later
+//                groupId PUBLISH_GROUP_ID
+//                artifactId PUBLISH_ARTIFACT_ID
+//                version PUBLISH_VERSION
+
+                // Two artifacts, the `aar` (or `jar`) and the sources
+                if (project.plugins.findPlugin("com.android.library")) {
+                    from components.release
+                } else {
+                    from components.java
+                }
+
+                artifact sourcesJar
+                artifact javadocJar
+
+                // Mostly self-explanatory metadata
+                pom {
+                    name = 'CA Mobile App Services'
+                    description = project.description
+                    url = siteUrl
+                    organization {
+                        name = 'CA API Management'
+                        url = 'https://github.com/CAAPIM'
+                    }
+                    licenses {
+                        license {
+                            name = 'MIT'
+                            url = 'https://github.com/CAAPIM/Android-MAS-SDK/blob/develop/LICENSE'
+                        }
+                    }
+
+                    scm {
+                        connection = gitUrl
+                        developerConnection = gitUrl
+                        url = siteUrl
+                    }
+                    issueManagement {
+                        system = 'github'
+                        url = issueUrl
+                    }
+                    developers {
+                        developer {
+                            id = 'apigatewayopensource'
+                            name = 'CA API Gateway Bot'
+                            email = 'API-Gateway-OpenSource@ca.com'
```

Comment:
is this still valid?

Reply:
This part has to be validated with Scott. Will conclude on this.

Reply:
i can't answer this. i sent question to Product team to verify. if they don't respond in time for your merge, i would say keep status quo and update later